theme/islandora-dept.tpl.php

```diff
@@ -9,8 +9,11 @@
 <?php if (isset($variables['tn'])): ?>
   <dl class="islandora-object-tn islandora-department">
     <dt>
-      <img src="<?php print $variables['tn']; ?>"/>
+      <img alt="<?php print $tn_alt;?>" id="<?php print $tn_id;?>" src="<?php print $variables['tn']; ?>"/>
```

Comment:
sorry @MorganDawe was about to merge and saw this..:( just for the sake of consistency, could you please change $tn_alt to $variables['tn_alt', etc?
I know they get auto-magically expanded, but makes reading, customizing later much easier if referenced from  $variables array.